time_tracker_api/activities/activities_namespace.py

```diff
@@ -41,41 +43,49 @@
     activity_response_fields
 )
 
-activities_dao = create_dao()
+activity_dao = create_dao()
 
 
 @ns.route('')
 class Activities(Resource):
     @ns.doc('list_activities')
-    @ns.marshal_list_with(activity, code=200)
+    @ns.marshal_list_with(activity, code=HTTPStatus.OK)
     def get(self):
-        return []
+        """List all activities"""
+        return activity_dao.get_all(), HTTPStatus.OK
 
     @ns.doc('create_activity')
+    @ns.response(HTTPStatus.BAD_REQUEST, 'Bad request')
     @ns.expect(activity_input)
-    @ns.marshal_with(activity, code=201)
-    @ns.response(400, 'Invalid format of the attributes of the activity.')
+    @ns.marshal_with(activity, code=HTTPStatus.CREATED)
     def post(self):
-        return ns.payload, 201
+        """Create an activity"""
+        return activity_dao.create(ns.payload), HTTPStatus.CREATED
 
 
 @ns.route('/<string:id>')
-@ns.response(404, 'Activity not found')
-@ns.param('id', 'The unique identifier of the activity')
+@ns.response(HTTPStatus.NOT_FOUND, 'Activity not found')
+@ns.param('id', 'The activity identifier')
 class Activity(Resource):
```

Comment:
Everytime the id is not well specified, like with an invalid format, lets say an string `asd`, you will get 
a 422 error. So lets specify
```
@ns.response(HTTPStatus.UNPROCESSABLE_ENTITY, 'The id has an invalid format')
```